sys-libs/libsepol/libsepol-3.9.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+inherit toolchain-funcs multilib-minimal
```

Comment:
Could you please fix https://bugs.gentoo.org/957978 with dot-a.eclass?
Because libsepol.a is required for libselinux and gcc has a new stable version, this will avoid many failures.